src/export/bops/index.ts

```diff
@@ -256,23 +254,7 @@ export function formatProposalDetails({
             ];
             const orderedContactItems: string[] = [];
             orderedContactKeys.forEach((key) => {
-              // If keysToRedact are present, redact "parts"
-              if (contactParts?.[key] && keysToRedact) {
-                const orderedContactKeysPassportMap = {
-                  title: `${fn}.title`,
-                  firstName: `${fn}.name.first`,
-                  lastName: `${fn}.name.last`,
-                  organisation: `${fn}.company.name`,
-                  phone: `${fn}.phone.primary`,
-                  email: `${fn}.email`,
-                };
```

Comment:
See the GitHub check that says `fn` is now unused since this block is gone - can we similarly delete the `const fn = ...` definition block?